specificatie/domain.yaml

```diff
@@ -570,12 +573,12 @@ components:
           type: "array"
           items:
             type: "string"
-        bijbehorendeAppartementsrechtIdentificaties:
+        bijbehorendeAppartementsrechten:
           description: |
-                      De kadstraalOnroerendeZaakidentificaties van de actuele appartementsrechten die bij deze KadastraalOnroerende Zaak (grondperceel) horen
+                      De verwijzingen naar de actuele appartementsrechten die bij deze KadastraalOnroerende Zaak (grondperceel) horen
           type : "array"
           items:
-            type: "string"
+            $ref: "#/components/schemas/BijbehorendAppartementsrecht"
```

Comment:
dit is toch een verwijzing naar een appartementsrecht? Toch niet naar een ander perceel (met een appartementsrecht)? Is dit niet hetzelfde probleem als de gerelateerden bij de natuurlijke persoon? Daar hebben we volgens mij best lang over  gediscussieerd toch? Of haal ik nu dingen door elkaar?

Reply:
Ja dit is een verwijzing naar de appartementsrechten die bij een perceel horen. 
Ik snap niet zo goed wat je bedoelt met "hetzelfde probleem als de gerelateerden bij de natuurlijke persoon?"

Verwijs je daarmee naar de BRP ? Daar hadden we met de complexiteit van de LO-GBA te maken en het feit dat de gegevens van gerelateerden (ouder, kinderen en partners) onderdeel van de persoonskaart van de ingeschreven persoon waren (en daarmee eigelijk ook eigenschappen van die persoon en niet alleen maar relaties naar andere personen ).

Volgens mij speelt hier niet een vergelijkbaar probleem.

Reply:
Het punt is: er zijn 4 soorten relaties tussen een kadastraal onroerende zaak en een andere kadastraal onroerende zaak, waarvan meerder instanties kunnne voorkomen (net zoals je ook meer dan 1 kind kan hebben). Echter, het lijkt nu alsof alle relaties bij alle kadastraal onroerende zaken kunnen voorkomen. Maar dat is niet zo, want een grondperceel heeft geen bijbehorend grondperceel. Heb je daarom gekozen voor bijbehorend appartementsrecht (en niet voor kadastraal onroerende zaak) omdat deze in dit geval niet van het type grondperceel kan zijn? 

Echter, nu krijg je het probleem dat een BijbehorendeAppartementsrecht geen resource is, maar ook geen entiteit in het model. Waar wordt hier dan naar verwezen?